driver/src/main/scala/api/collections/UpdateOps.scala

```diff
@@ -194,20 +195,11 @@ trait UpdateOps[P <: SerializationPack] extends UpdateCommand[P]
 
     /** The max BSON size, including the size of command envelope */
     private def maxBsonSize = {
-      val builder = pack.newBuilder
-      val emptyElm = new UpdateElement(
-        q = builder.document(Seq.empty),
-        u = builder.document(Seq.empty),
-        upsert = false,
-        multi = false,
-        collation = None,
-        arrayFilters = Seq.empty)
-
       // Command envelope to compute accurate BSON size limit
       val emptyCmd = new ResolvedCollectionCommand(
         collection.name,
         new Update(
-          emptyElm, Seq.empty, ordered, writeConcern, false))
```

Comment:
Doesn't make sense to me.
Minimal bulk size should contain at least one empty document.

Reply:
Reverted. Updated the fix to minimise the changes.